### ISSUE TITLE
Community Contribution: Integrate unplugin-auto-import for Vitest

### ISSUE THREAD

**Issue by rainerhahnekamp** [softarc-consulting/sheriff#127]
**This issue is meant to be implemented by the community. Please write a comment if you want to get this issue assigned.**

Instead of using global types or always import the vitest commands, automatically, migrate the tests to use `AutoImport`: https://github.com/unplugin/unplugin-auto-import

Instructions on how to do that with Vitest are available at: https://vitest.dev/config/#globals

**Comment by bulicovic** [comment]
Hey @rainerhahnekamp is this issue still valid? I would like to contribute :)

**Comment by rainerhahnekamp** [comment]
Hey @bulicovic - thanks for the interest! With Angular 21 making Vitest the default, the original goal of this issue is likely obsolete. I’d rather not have you work on something that will disappear soon, so I’m inclined to close this.

That said, we’d really appreciate your help on other parts of the project. I can prepare or point you to a few good first issues that are still relevant - would you be up for that?

**Comment by bulicovic** [comment]
@rainerhahnekamp yes, I already suspected that it is potentially obsolete. I would like to help and contribute :)

**Comment by rainerhahnekamp** [comment]
Here's a list of issues which might be good to start. Let me know which one you are interested and I can come up with more details:
- #151 
- #166 
- #21

**Comment by bulicovic** [comment]
I can start with the first one: https://github.com/softarc-consulting/sheriff/issues/151

**Comment by rainerhahnekamp** [comment]
I’ve added an explanation to #151. please make a comment over there, so that I can assign the issue to you

**Comment by rainerhahnekamp** [comment]
Closing as discussed
